src/middlewared/middlewared/plugins/disk.py

```diff
@@ -34,6 +34,10 @@
 RE_MPATH_NAME = re.compile(r'[a-z]+(\d+)')
 RE_SED_RDLOCK_EN = re.compile(r'(RLKEna = Y|ReadLockEnabled:\s*1)', re.M)
 RE_SED_WRLOCK_EN = re.compile(r'(WLKEna = Y|WriteLockEnabled:\s*1)', re.M)
+RAWTYPE = {
+    'freebsd-zfs':  '516e7cba-6ecf-11d6-8ff8-00022d09712b',
+    'freebsd-swap': '516e7cb5-6ecf-11d6-8ff8-00022d09712b',
+}
```

Comment:
The linter flagged the 'freebsd-zfs' line because I aligned the ID. Is that worth changing?

Reply:
Yes, thats not valid PEP8.